platformcomponents/mobile/popoverPage.json

```diff
@@ -0,0 +1,12 @@
+{
+  "popoverPage": {
+    "comment": "Applied to popover pages like the ones appearing next to sliderScrollbar",
+    "figma": "https://www.figma.com/file/L67wWWZaXcvVOLEpn77adE/Components---iOS?node-id=17907%3A154840",
+    "background": {
+      "#normal": ""
```

Comment:
not quite sure if this should be merged if there's no color set. Maybe just add this once a color is set.

Reply:
nono, this was only temporary because the token was not available